activerecord/lib/active_record/enum.rb

```diff
@@ -119,8 +132,9 @@ def serialize(value)
       attr_reader :name, :mapping
     end
 
-    def enum(definitions)
+    def enum(definitions, options = {})
```

Comment:
yeah I agree, but first hash is current api =(
Sure, I can change it to keyword arguments.